api/resources/component.go

```diff
@@ -64,7 +64,7 @@ type CPUQuantity struct {
 
 func (c *CPUQuantity) MarshalJSON() ([]byte, error) {
 	capInStr := strconv.FormatInt(c.MilliValue(), 10)
-	return []byte(fmt.Sprintf(`"%s"`, capInStr)), nil
```

Comment:
discussed with @wlchn and we think it's better to suffix cpuRequest with 'm'.

otherwise, if use input: 1 for  `cpuRequest`, in response it will be: `"cpuRequest": "1000"`, this might cause confusion. 

@davidqhr plz check